tox.ini

```diff
@@ -87,27 +94,17 @@ commands =
     check-manifest {toxinidir}
     flake8 src tests setup.py
 
-[testenv:coveralls]
+[testenv:coverage]
 deps =
-    coveralls
-skip_install = true
-commands =
-    coveralls []
-
-[testenv:codecov]
```

Comment:
Did you drop the codecov support?

Reply:
No. This method is deprecated. It is now integrated in GitHub Actions like so: https://github.com/melexis/warnings-plugin/pull/112/files#diff-ee49282f461b4c8ad179f79dd5bcdf93124561074c64a771366caf93e99b9320R34

https://app.codecov.io/gh/melexis/warnings-plugin/commits?page=1